lang/src/org/partiql/lang/ast/passes/AstVisitor.kt

```diff
@@ -27,7 +27,7 @@ import org.partiql.lang.ast.*
  *
  *  One `visit*` function is included for each base type in the AST.
  */
-@Deprecated("Use AstNode#iterator() or AstNode#children()")
+@Deprecated("Use org.lang.partiql.domains.PartiqlAst.Visitor instead")
```

Comment:
Similar to comment in `ast.kt`. Should this be included since `PartiqlAst` isn't part of the v0.1.* branches?

Reply:
same response as above.